benches/go/cmd/sweep/main.go

```
@@ -0,0 +1,168 @@
+)
+
+// The one command line every row is measured against, the same one the Rust shadows
+// use, so the two cards describe the same work.
+var words = []string{"use", "-g", "node@20"}
+
+// Rounds, and per-round iteration counts chosen so a round is ~0.5-3ms of work.
+const rounds = 4000
+
+// sink keeps a parse from being optimized away. Go has no `black_box`, and a compiler
+// that can see the result is unused is within its rights to skip producing it.
+var sink bool
```

golangci-lint (2.12.2)

[error] var sink is unused

(unused)